src/components/search/SearchVideoCard.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import cardFallbackImg from '@edx/brand/paragon/images/card-imagecap-fallback.png';
+import { Link } from 'react-router-dom';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+import {
+  Card, Chip, Icon, Truncate,
+} from '@openedx/paragon';
+
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { PlayCircleOutline } from '@openedx/paragon/icons';
+import { useEnterpriseCustomer } from '../app/data';
+
+const SearchVideoCard = ({
+  key, hit, isLoading, parentRoute, ...rest
+}) => {
+  const { data: enterpriseCustomer } = useEnterpriseCustomer();
+
+  const video = useMemo(
+    () => {
+      if (!hit) {
+        return {};
+      }
+      return camelCaseObject(hit);
+    },
+    [hit],
+  );
+  const linkToVideo = useMemo(
+    () => {
+      if (!Object.keys(video).length) {
+        return '#';
+      }
+      return `/${enterpriseCustomer.slug}/videos/${video?.aggregationKey}/`;
```

Comment:
We need `edxVideoId` here in the url, are `aggregationKey` and `edxVideoId` same?

Reply:
yes, both are same.